services/context/context.go

```diff
@@ -102,6 +102,18 @@ func NewTemplateContextForWeb(ctx *Context) TemplateContext {
 	tmplCtx["Locale"] = ctx.Base.Locale
 	tmplCtx["AvatarUtils"] = templates.NewAvatarUtils(ctx)
 	tmplCtx["RootData"] = ctx.Data
+	tmplCtx["Consts"] = map[string]any{
```

Comment:
I have thought about your solution vs `tmplCtx["Consts"]`, I chose the second one because: there will be more constants to be injected to templates, I do not think it is good to keep bloating the template context map.

And "Units" is not easy to search, at least it should be "UnitConsts", but I think "Const" is better at the moment.